src/gp_activetable.c

```diff
@@ -51,7 +51,13 @@ typedef struct DiskQuotaSetOFCache
 	HASH_SEQ_STATUS pos;
 } DiskQuotaSetOFCache;
 
-HTAB *active_tables_map = NULL; // Set<DiskQuotaActiveTableFileEntry>
+HTAB       *active_tables_map                  = NULL; // Set<DiskQuotaActiveTableFileEntry>
+TimestampTz active_tables_last_overflow_report = 0;
+
+#define ACTIVE_TABLES_MAP_WARNING                                                 \
+	"[diskquota] the number of active tables reached the limit, please increase " \
+	"the GUC value for diskquota.max_active_tables. Current "                     \
+	"diskquota.max_active_tables value:"
```

Comment:
I think that the current value in the warning can be omitted; this will greatly simplify the functions.